src/refresh.js

```diff
@@ -0,0 +1,25 @@
+import { checkForAlpine } from './utils'
+
+const AlpineRefreshMagicMethod = {
+    start() {
+        checkForAlpine()
+
+        Alpine.addMagicProperty('refresh', ($el) => {
+            if (!$el.__x) {
+                return () => {}
+            }
+
+            return () => $el.__x.updateElements($el)
```

Comment:
I think we can do:

```js
return (el = $el) => el.__x.updateElements(el)
```

and get it working with the `$component` helper after too after #76

Reply:
Ah, good shout. Hadn't though about that!